### PR TITLE
docs: reflect v2.3 release changes in roadmap.md

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

PR reflects v2.3 release changes in roadmap.md .

### CODE REVIEW

**Comment by alexmt** [comment]
Based on discussing in contributor meetings I've moved several items to v2.4.  @wtam2018 , @leoluz , @crenshaw-dev  let me know please if you want to add/remove something from v2.4 plans.

File: docs/roadmap.md
Comment (jannfis) [inline docs/roadmap.md:18]: I don't want to push this one out much further, to be honest.

Comment (alexmt) [reply docs/roadmap.md:18]: I know it took forever to make progress here. Would you change your mind if we prioritize it and started working on it in e.g. next release? Meaning you won't not have to chase maintainers and wait for feedback on proposal? :)

File: docs/roadmap.md
Comment (ishitasequeira) [inline docs/roadmap.md:8]: As we are considering to add this as part of `[Support multiple sources for an Application](#support-multiple-sources-for-an-application)`, should we move this to `v2.5` as well?

Comment (ishitasequeira) [inline docs/roadmap.md:8]: As we are considering to add this as part of [Support multiple sources for an Application](#support-multiple-sources-for-an-application), should we move this to v2.5 as well?

Comment (ishitasequeira) [inline docs/roadmap.md:8]: As we are considering to add this as part of `[Support multiple sources for an Application](#support-multiple-sources-for-an-application)`, should we move this to v2.5 as well?

Comment (alexmt) [reply docs/roadmap.md:8]: Agree. I forgot about this feature. Moved to it.

File: docs/roadmap.md
Comment (alexmt) [reply docs/roadmap.md]: Sounds good. I've mentioned both UI/CLI and security enhancements in this roadmap item. Let me know what do you think please

Comment (leoluz) [reply docs/roadmap.md]: Looks good. Tks!

File: docs/roadmap.md
Comment (crenshaw-dev) [inline docs/roadmap.md:10]: I think parameterization and security should target 2.4 but UI/CLI integration should wait for 2.5.

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

**Review by crenshaw-dev — APPROVED** [review]

**Pull request merged**
